cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/ColumnActionsDropdown.scss

```diff
@@ -58,6 +63,7 @@
       font-weight: 400;
       line-height: 2;
       cursor: initial;
+      box-shadow: 1px 1px #cccccc;
```

Comment:
We might need it for the second level popover menu too if its on top of the right panel (columns/directive tab).